static/js/compose_height.js

```diff
@@ -40,8 +29,8 @@ export function make_compose_box_full_size() {
 export function make_compose_box_original_size() {
     set_is_full_size(false);
 
-    $("#compose-textarea").css("max-height", "calc(50vh - 50px)");
-    $("#floating_recipient_bar").css("z-index", 98);
+    $("#compose").removeClass("compose-fullscreen");
+    $("#floating_recipient_bar").show();
```

Comment:
Could we replace the `floating_recipient_bar` show/hide logic with just having the `compose-fullscreen` class set a z-index 1 higher than that for the `floating_recipient_bar`?

Reply:
Yes, I will try to also find why I did so.